vector/src/main/res/layout/activity_tchap_room_creation.xml

```diff
@@ -110,10 +110,41 @@
                 android:layout_marginTop="13dp"
                 android:layout_marginLeft="10dp"
                 android:layout_marginRight="7dp"
-                android:text="@string/tchap_room_creation_public_private_info"
+                android:text="@string/tchap_room_creation_public_info"
                 android:textSize="12sp"
+                android:paddingBottom="10dp"
                 android:textColor="@color/vector_tchap_text_color_light_grey"
                 tools:text="Un salon public peut être rejoint par n'importe quel membre de la communauté Tchap, il ne doit contenir aucune donnée sensible."/>
```

Comment:
this `tools:text` looks a bit useless

Reply:
why? I think this is better than see "@String/xxx" in preview

Reply:
Let's talk about it IRL :)